vendor/engines/secure_rooms/app/services/secure_rooms/access_rules/valid_account_rule.rb

```diff
@@ -0,0 +1,15 @@
+module SecureRooms
+
+  module AccessRules
+
+    class ValidAccountRule < BaseRule
+
+      def evaluate
+        deny! "User has no valid accounts for this Product" if @accounts.blank?
```

Comment:
Not a big deal, but would it be simpler/clearer if this were added into the existing account checking class?